R/helper_clean_model_data.R

```diff
@@ -107,7 +107,7 @@ clean_model_data <- function(formula,
 
     if (any(ret[["condition_pr"]] <= 0 | ret[["condition_pr"]] > 1)) {
       stop(
-        "`condition_pr` must be a vector of positive values no greater than 1"
+        "`condition_prs` must be a vector of positive values no greater than 1"
```

Comment:
does not match the code ??? formal argument does not have the s

Reply:
that's because clean_model_data doesn't use plural. It should. The `condition_prs` matches the argument users see.

Reply:
I don't think I want to fix this right now? It's all internal (but problematic).